src/main/webapp/WEB-INF/tags/instructor/courseStudentDetailsEdit/studentInformationTable.tag

```diff
@@ -4,11 +4,14 @@
 <%@ taglib tagdir="/WEB-INF/tags" prefix="t" %>
 <%@ attribute name="studentInfoTable" type="teammates.ui.template.StudentInfoTable" required="true" %>
 <%@ attribute name="newEmail" required="true" %>
+<%@ attribute name="anyEmailSentForTheCourse" required="true" %>
 <%@ tag import="teammates.common.util.Const" %>
 <div class="panel panel-primary">
     <div class="panel-body fill-plain">
         <form action="<%=Const.ActionURIs.INSTRUCTOR_COURSE_STUDENT_DETAILS_EDIT_SAVE%>" method="post" id="instructor-student-edit-form" class="form form-horizontal">
             <input type="hidden" name="<%=Const.ParamsNames.COURSE_ID%>" value="${studentInfoTable.course}">
+            <input type="hidden" id="<%=Const.ParamsNames.EMAIL_SEND_CHECK%>" name="<%=Const.ParamsNames.EMAIL_SEND_CHECK%>" value="false">
+            <input type="hidden" id="isAnyEmailSentForTheCourse" name="isAnyEmailSentForTheCourse" value="${anyEmailSentForTheCourse}">
```

Comment:
Is this `input` used to pass the true/false value to the server? If not, whatever's stored here can be moved to javascript instead.